super_csv/serializers.py

```diff
@@ -38,5 +40,15 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        if bool(operation.data):
+            data = json.load(operation.data)
+            return CSVOperationDataSerializer(data).data
+        else:
+            error_message = 'Failed to retreive file'
+            logger.info(_(error_message))
```

Comment:
I don't think we usually translate our log messages, and it is failing a quality check. I think it's possible that there's other bad data besides just file so I'd update it to be more generic (with fixed spelling on "retrieve"). 

```suggestion
            logger.info('Failed to retrieve CSV operation data.')
```